src/components/calcite-pagination/calcite-pagination.tsx

```diff
@@ -0,0 +1,216 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Prop,
+  Method,
+  State,
+  Watch
+} from "@stencil/core";
+import { getElementDir } from "../../utils/dom";
+import { CSS, TEXT } from "./resources";
+import { chevronLeft16, chevronRight16, ellipsis16 } from "@esri/calcite-ui-icons";
+import CalciteIcon from "../../utils/CalciteIcon";
+
+const maxPagesDisplayed = 5;
```

Comment:
Could this be user configurable at some point?

Reply:
Yea I think this could be a future enhancement.